packages/engine/src/avatar/AvatarInputSchema.ts

```diff
@@ -553,7 +556,7 @@ export const createAvatarInput = () => {
   map.set(GamepadAxis.Left, BaseInput.MOVEMENT_PLAYERONE)
   map.set(GamepadAxis.Right, BaseInput.GAMEPAD_STICK_RIGHT)
 
-  if (XRUserSettings.invertRotationAndMoveSticks) {
+  if (avatarInputState.invertRotationAndMoveSticks.value) {
```

Comment:
this will not work, as `createAvatarInput` is called once at engine startup, this was not properly implemented previous. to have this be dynamic, we need to manually update the mapping when the `invertRotationAndMoveSticks` changes - we can use an engine action receptor for this, we should put it in the AvatarControllerSystem